sample/https-client.c

```diff
@@ -275,7 +275,7 @@ main(int argc, char **argv)
 	}
 
 	path = evhttp_uri_get_path(http_uri);
-	if (path == NULL) {
+	if (path == NULL || strlen(path) == 0) {
```

Comment:
Here you don't need to check `path == NULL` at all, since `evhttp_uri_parse()` will always fill `->path` when it not fails (IOW it return not NULL, and here we are checking this)